cmd/mesh/upgrade.go

```diff
@@ -50,8 +50,8 @@ type upgradeArgs struct {
 	context string
 	// wait is flag that indicates whether to wait resources ready before exiting.
 	wait bool
-	// yes means skipping the prompting confirmation for value changes in this upgrade.
-	yes bool
+	// skipConfirmation means skipping the prompting confirmation for value changes in this upgrade.
+	skipConfirmation bool
```

Comment:
please rebase to remove these changes.

Reply:
This is different from manifest's --yes flag. I've rebased 1 hour ago..

Reply:
I include the skipConfirmation renaming into this PR (I once thought it would be small).

Reply:
oh ok, so this was not done in the original renaming pr? if so this is ok.

Reply:
you are right, not done before this PR.